client/src/components/shared/Layout/LayoutTopBarCalendar/Calendar/Calendar.tsx

```diff
@@ -31,11 +34,18 @@ const Calendar = (): ReactElement => {
           isAfter(curr.from, currentDate) &&
           !acc.some(element => element.isActive));
 
-      acc.push({ ...curr, isActive });
+      acc.push({
+        ...curr,
+        isActive,
+        isAlert:
+          isActive &&
+          curr.label === i18n.t('views.projects.projectsTimelineWidget.allocationWindow') &&
```

Comment:
This looks very fragile. It's hard ot remember about full-path i18n keys when changing them. Can we rely on some flag instead?